Cargo.toml

```diff
@@ -32,7 +32,8 @@ regex = "1"
 serde = "1.0"
 serde_derive = "1.0"
 thiserror = "1"
-tokio = { version = "1.0", features = [ "sync", "time" ] }
+tokio = { version = "1", features = [ "sync", "time" ] }
```

Comment:
I don't know why it works before, but it looks that the `rt` feature is needed. Because of `tokio::spawn`.